commands/create/cluster/command.go

```diff
@@ -27,12 +27,13 @@ import (
 // Arguments contains all possible input parameter needed
 // (and optionally available) for creating a cluster.
 type Arguments struct {
-	APIEndpoint              string
-	AuthToken                string
-	AvailabilityZones        int
-	ClusterName              string
-	DryRun                   bool
-	FileSystem               afero.Fs
+	APIEndpoint       string
+	AuthToken         string
+	AvailabilityZones int
+	ClusterName       string
+	DryRun            bool
+	// fileSystem is not exposed to hide it from comparison in tests.
+	fileSystem               afero.Fs
```

Comment:
I think you want to use https://godoc.org/github.com/google/go-cmp/cmp/cmpopts#IgnoreFields.

Reply:
check - thanks